csp_billing_adapter/bill_utils.py

```diff
@@ -58,7 +78,28 @@ def get_billable_usage(
     usage_records: list,
     config: Config,
     empty_usage: False
-):
+) -> dict:
+    """
+    Processes the provided 'usage_records' to determine the billable
+    usage details for the metrics specified in the 'config', using
+    the aggregation method defined for the metric in the config, and
+    returns a hash containing the determined usage details for each
+    specified metric. The 'usage_records' list can be optionally
+    cleared after processing, indicated by the 'empty_usage' flag.
```

Comment:
This one is a little confusing. empty_usage is to say that we want to get a list of metrics that would be billed and only bill 0. This is because we have to bill every hour even if we only bill usage on a monthly basis. So it does not correlate to emptying the records necessarily.

Reply:
Something like: a list of billing metrics with usage of zero can be provided using the empty_usage flag.

Reply:
Thx, yeah I think I understand that stuff better after having done the later functions... Should have gone back and cleaned that explanation up a bit more.